pkg/pf/tests/testdata/TestDetailedDiffList/attribute_no_replace/added_end.golden

```diff
@@ -35,8 +35,6 @@ Plan: 0 to add, 1 to change, 0 to destroy.
         [id=test-id]
         [urn=urn:pulumi:test::project::testprovider:index/test:Test::p]
       ~ keys: [
-            [0]: "val1"
-            [1]: "val2"
```

Comment:
This is interesting. Looks like a fix, it figured something out better? Why was the previous value like this? Sounds like there's an opportunity to make Pulumi default detailed differ better yes?